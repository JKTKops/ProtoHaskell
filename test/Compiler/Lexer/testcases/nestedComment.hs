module

{- {- some nested content
-}
other content
-}

foo bar
