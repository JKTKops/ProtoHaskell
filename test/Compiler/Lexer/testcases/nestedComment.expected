Right [<indentation>, module, <indentation>, foo, bar]